fe/fe-core/src/main/java/org/apache/doris/planner/PlanNode.java

```diff
@@ -133,7 +134,7 @@ abstract public class PlanNode extends TreeNode<PlanNode> {
 
     protected List<SlotId> outputSlotIds;
 
-    private NodeType nodeType = NodeType.DEFAULT;
+    protected NodeType nodeType = NodeType.DEFAULT;
     protected StatsDeriveResult statsDeriveResult = new StatsDeriveResult();
```

Comment:
Before entering statistics derivation, ```StatsDeriveResult``` should be null. It should not be set directly in the constructor.

Reply:
When the derivation of statistics for all nodes is complete, the initialization needs to be removed.
The reason for the current increase is that some uts cannot pass